qemu/tests/virtual_nic_stress.py

```diff
@@ -92,11 +91,12 @@ def unload_stress(session):
                           " Check if the network is still alive",
                           logging.info)
     count = params["count"]
-    timeout = float(count) * 2
-    status, output = utils_net.ping(host_ip, count, session=session,
-                                    timeout=timeout)
-    if status != 0:
-        test.fail("Ping failed, status: %s,"
-                  " output: %s" % (status, output))
-
+    count_cmd = "ping %s -c %s" % (guest_ip, count)
+    try:
+        output = process.system_output(count_cmd, timeout=float(count) * 2)
+    except process.CmdError:
+        err = "Ping from host to guest failed"
+        err += "Command: %s," % count_cmd
+        err += "output: %s." % output
+        test.fial(err)
```

Comment:
typo

Reply:
@kimi1978wy Thanks for your review, I have changed them. Please help me review it again.